get_download_test.py

```diff
@@ -0,0 +1,37 @@
+from selenium import webdriver

```

Comment:
Let's try to change this using an internal library like `requests` - I've had several issues running selenium on a webserver where this is going to ultimately run.

Reply:
Gotcha. Will find a way to do it without selenium.

Reply:
Good news is it looks like the URL changes on a consistent pattern (every second Monday of the month) so `requests` should be able to work for this.

Reply:
Did you want this program constantly running in the program or something you use intermittently?

Reply:
This is something that will be run weekly or monthly. This is a great start that can get fleshed out. We will worry about the schedule and validation later.